querybook/webapp/components/DataTableView/DataTableView.tsx

```diff
@@ -176,6 +178,17 @@ export const DataTableView: React.FC<IDataTableViewProps> = ({ tableId }) => {
     });
 
     useTrackView(ComponentType.TABLE_DETAIL_VIEW);
+    const triggerSurvey = useSurveyTrigger(true);
+    useEffect(() => {
+        if (!tableId || !tableName) {
+            return;
+        }
+        triggerSurvey(SurveySurfaceType.TABLE_TRUST, {
+            table_id: tableId,
+            table_name: tableName,
```

Comment:
also log `schemaName`? or this should the full table name?
table_id may be not needed.

Reply:
table_name is schema name